docs/source/cli.mdx

```diff
@@ -0,0 +1,53 @@
+# Command Line Interface (CLI)
+
+🤗 Datasets provides a command line interface (CLI) with useful shell commands to interact with your dataset.
+
+You can check the available commands:
+```shell
+$ datasets-cli --help
+usage: datasets-cli <command> [<args>]
+
+positional arguments:
+  {convert,env,test,run_beam,dummy_data,convert_to_parquet}
+                        datasets-cli command helpers
+    convert             Convert a TensorFlow Datasets dataset to a HuggingFace Datasets dataset.
+    env                 Print relevant system environment info.
+    test                Test dataset implementation.
+    run_beam            Run a Beam dataset processing pipeline
+    dummy_data          Generate dummy data.
+    convert_to_parquet  Convert dataset to Parquet
+
+optional arguments:
+  -h, --help            show this help message and exit
+```
+
+## Convert to Parquet
+
+Easily convert your Hub [script-based dataset](dataset_script) to Parquet [data-only dataset](repository_structure), so that the dataset viewer will be supported.
+
+```shell
+$ datasets-cli convert_to_parquet --help
+usage: datasets-cli <command> [<args>] convert_to_parquet [-h] [--token TOKEN] [--revision REVISION] [--trust_remote_code] dataset_id
+
+positional arguments:
+  dataset_id           source dataset ID, e.g. USERNAME/DATASET_NAME or ORGANIZATION/DATASET_NAME
+
+optional arguments:
+  -h, --help           show this help message and exit
+  --token TOKEN        access token to the Hugging Face Hub
+  --revision REVISION  source revision
+  --trust_remote_code  whether to trust the code execution of the load script
+```
+
+This command:
+- makes a copy of the script on the "main" branch into a dedicated branch called "script" (if it does not already exists)
+- creates a pull request to the Hub dataset to convert it to Parquet files (and deletes the script from the main branch)
+
+If in the future you need to recreate the Parquet files from the "script" branch, pass the `--revision script` argument.
+
+Note that you should pass the `--trust_remote_code` argument only if you trust the remote code to be executed locally on your machine.
+
+For example:
+```shell
+$ datasets-cli convert_to_parquet USERNAME/DATASET_NAME --token USER_ACCESS_TOKEN
```

Comment:
Let's just remove the `$`

```suggestion
datasets-cli convert_to_parquet USERNAME/DATASET_NAME --token USER_ACCESS_TOKEN
```

Reply:
OK, I added to make clear what the input was (and the output, without leading dollar sign). But I will remove it then.

Reply:
Is it OK if I use `>>>`, as huggingface-hub docs do? https://huggingface.co/docs/huggingface_hub/en/guides/cli

That way, in the examples above (with input and output), the copy/paste just copies the input. If I remove any leading symbol, the copy/paste will copy input and output...

Reply:
Hmmm, it seems like a bug to use `>>>` for cli @Wauplin 

In our case we have no output, so I would just remove `$`

Reply:
We have output in the 2 code snippets above the third on.

Reply:
Ah sure. Hmmm, let's use `>>>` then and see the alternative in another PR maybe

Reply:
See:
- https://github.com/huggingface/datasets/pull/6831/files#diff-25d866ca4099e701561fffdb962611aedbc2598e7fc12975831cc614277f10f2R6-R22
- https://github.com/huggingface/datasets/pull/6831/files#diff-25d866ca4099e701561fffdb962611aedbc2598e7fc12975831cc614277f10f2R28-R40

Reply:
> Hmmm, it seems like a bug to use >>> for cli

> Ah sure. Hmmm, let's use >>> then and see the alternative in another PR maybe

Happy to align to whatever convention we want. I haven't really thought the it TBH

Reply:
The same for me.